tests/unit/test_install.py

```diff
@@ -29,10 +34,42 @@
 from databricks.labs.ucx.config import WorkspaceConfig
 from databricks.labs.ucx.framework.dashboards import DashboardFromFiles
 from databricks.labs.ucx.framework.install_state import InstallState
+from databricks.labs.ucx.framework.tasks import Task
 from databricks.labs.ucx.install import WorkspaceInstaller
 
 from ..unit.framework.mocks import MockBackend
 
+# test cluster id
+cluster_id = "9999-999999-abcdefgh"
+
+
+def mock_clusters():
+    from databricks.sdk.service.compute import DataSecurityMode, State
+
+    return [
+        MagicMock(
```

Comment:
```suggestion
        compute.ClusterInfo(
```

we have a concrete type, use this one

Reply:
Done